src/scripts.js

```diff
@@ -3,8 +3,11 @@
 
 import './styles.css'
 import { closeRecipe, showRecipe, switchView, searchForRecipes, updateUserRecipes, toggleTagData, renderActiveTag, displayTaggedRecipes, updateRecipesFromModal } from './domUpdates';
-import { calculateRecipeCost, getIngredientAmounts, getInstructions } from './recipes'; 
-import './images/antipasti.png';
+import { calculateRecipeCost, getIngredientAmounts, getInstructions } from './recipes';
+import './images/appear-bookmark-icon.png'
+import './images/hover-bookmark-icon.png'
+import './images/select-bookmark-icon.png'
+import './images/antipasti.png'
```

Comment:
do these need to be imported here? Actually it looks like none of these images need to be imported here anymore?

Reply:
It was asked for in the original document. I would not want to change that due to the approaching deadline. Can remove the unused pictures.